example/Page_load.html

```diff
@@ -0,0 +1,17 @@
+  <h1> This sample code will help you to open the chat widget automatically on page load</h1>
+
+
+< script type = "text/javascript" >
+	(function (d, m) {
+		var kommunicateSettings = {
+    "appId": "kommunicate-support","popupWidget": true,"automaticChatOpenOnNavigation": true,
```

Comment:
@harishbala1o add instructions to replace "kommunicate-support" with the customer APP_ID from install section 

all format the code